pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -4702,37 +4701,38 @@ protected void internalGetReplicatedSubscriptionStatus(AsyncResponse asyncRespon
         }
     }
 
-    private void internalGetReplicatedSubscriptionStatusForNonPartitionedTopic(AsyncResponse asyncResponse,
+    private CompletableFuture<Void> internalGetReplicatedSubscriptionStatusForNonPartitionedTopicAsync(
```

Comment:
No need to change the return type.
Do not add "Async", let's keep the method name consistent with others.